include/sound/sof/dai.h

```diff
@@ -85,4 +85,14 @@ struct sof_ipc_dai_config {
 	};
 } __packed;
 
+/* general purpose DAI clock control */
+struct sof_ipc_dai_clkctrl {
+	struct sof_ipc_cmd_hdr hdr;
+	uint32_t type;		/**< DAI type - enum sof_ipc_dai_type */
+	uint32_t dai_index;	/**< index of this type dai */
+
+	uint32_t clk_en;	/**< zero to disable the clock */
+	uint32_t clk_id;	/**< clock id to operate */
```

Comment:
how sure are we that this is all we'll ever need? Can it be that at some point we'll also need to send a rate down? Or maybe even the duty cycle (ok, maybe not this one)?